### PR TITLE
Opcda and 32 bit Python

### PR DESCRIPTION
**Pull request by coolafabbe**
Pull request for OPC DA and convert to Python 3.8.10 - 32 bit 
Driver for OPC DA is in "development"-driver

### CODE REVIEW

File: requirements.linux [hunk 1]
```diff
@@ -3,5 +3,5 @@ opcua==0.98.12
 SimpleWebSocketServer==0.1.1
 paho-mqtt==1.5.1
 pyads==3.3.8
-fastcrc==0.1.1
+crccheck==0.1.1
```
Comment (AHugoson) [inline requirements.linux]: I think this should be `crccheck==1.1`?
(like in the windows requirements)

File: requirements.mac [hunk 2]
```diff
@@ -3,5 +3,5 @@ opcua==0.98.12
 SimpleWebSocketServer==0.1.1
 paho-mqtt==1.5.1
 pyads==3.3.8
-fastcrc=0.1.1
+crccheck=0.1.1
```
Comment (AHugoson) [inline requirements.mac]: I think this should be `crccheck==1.1`?
(like in the windows requirements)

File: src/drivers/opcda_client/opcda_client.py [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+from ..driver import VariableQuality, driver
+
+import OpenOPC
+import pywintypes # To avoid timeout error
+pywintypes.datetime=pywintypes.TimeType
+
+class opcda_client(driver):
+    '''
+    Driver that can be used for development. The driver can be used on a component just assigning the driver type "development".
```
Comment (AHugoson) [inline src/drivers/opcda_client/opcda_client.py]: this needs an update

**Review by AHugoson — CHANGES_REQUESTED** [review]
Just some small notes. The description of the opcda_client class is not updated, and the requirements had a version number that I think does not exist?

File: src/drivers/opcda_client/opcda_client.py (hunk 3 above)
Comment (MikelAyani) [reply src/drivers/opcda_client/opcda_client.py]: Better?

**Review by AHugoson — APPROVED** [review]

File: src/drivers/opcda_client/opcda_client.py (hunk 3 above)
Comment (AHugoson) [reply src/drivers/opcda_client/opcda_client.py]: Perfect ;)

**Pull request merged**